plugins/binaryninja/justfile

```
@@ -2,15 +2,15 @@ init:
 	mkdir -p {{justfile_directory()}}/mui/common_resources
 	cp {{justfile_directory()}}/../common/* {{justfile_directory()}}/mui/common_resources/
 	test -d venv || python3 -m venv venv
-	. venv/bin/activate; pip install -r requirements-dev.txt -r binjastub/requirements.txt
+	. venv/bin/activate; pip3 install -r requirements-dev.txt -r binjastub/requirements.txt
```

Using plain `pip` should always be right given that we've activated the virtual environment and things are set up correctly for us, but I think a better change would be
```suggestion
	. venv/bin/activate; python3 -m pip install -r requirements-dev.txt -r binjastub/requirements.txt
```